CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # 2020-01-08
 
+## API changes (relevant client developers)
+
+- Allow up to 256 characters as handle, dots and dashes too (#953)
+  - All handles related endpoints, namely:
+    - POST "/users/handles"
+    - HEAD "/users/handles/:handle"
+    - GET "/users/handles/:handle"
+  - now accept this new format of handles
+- Refuse to delete non-empty IdPs (412 precondition failed) (#875)
+  - DELETE "identity-providers/:idp" will now return 412 if there are users provisioned with that IDP
+- Linear onboarding feature: Provide information about custom backends (#946)
+  - New public endpoint:
+    - GET "/custom-backend/by-domain/:domain"
+  - New interal endpoints:
+    - PUT "/i/custom-backend/by-domain/:domain"
+    - DELETE "/i/custom-backend/by-domain/:domain"
```

Comment:
```suggestion
```

There is only a PUT endpoint, no delete endpoint. See https://github.com/wireapp/wire-server/pull/960/files#diff-1682ecde5b570fb15dfaa9ccbeb59a5cR75

Reply:
Code says otherwise :)

develop:

https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/API.hs#L1078

this PR:
https://github.com/wireapp/wire-server/blob/1795e7d6ea15c41f136b0fa24daaddbe33d39603/services/galley/src/Galley/API.hs#L1078

Reply:
https://github.com/wireapp/wire-server/pull/960/files#diff-1682ecde5b570fb15dfaa9ccbeb59a5cR75 is about config.json, https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/API.hs#L1078 is about the domain-based redirect *to* that config.json.

so you're both right.  :)